opm/input/eclipse/Schedule/Schedule.cpp

```diff
@@ -136,6 +136,9 @@ namespace Opm {
         this->restart_output.resize(this->m_sched_deck.size());
         this->restart_output.clearRemainingEvents(0);
 
+        auto simUpdate = new SimulatorUpdate();
+        this->simUpdateFromPython = simUpdate;
```

Comment:
I hope this can me made with `make_...` if we are able to use something that is not a raw pointer.